src/components/Editor/index.tsx

```diff
@@ -92,18 +92,6 @@ export default function Editor(props: { docKey: string }) {
     };
   }, [docKey, client, doc]);
 
-  if (!client || !doc) {
-    return null;
-  }
```

Comment:
The reason for removing this line is that the client and doc object always exist when loading is false.